### ISSUE TITLE
"Joining instructions" in Google Calendar are misleading

### ISSUE THREAD

**Issue by YodaLightsabr** [hackclub/slash-z#188]
Google Calendar scheduled links include a "Joining instructions" button.

<img src="https://github.com/hackclub/slash-z/assets/76178582/a91fc0db-efad-4107-91cf-03a6d892a5d2" width=500>

This button looks like you'll be able to click it to get instructions to share with someone. This makes sense if you're running a larger event where not everyone will be added to the calendar invite, such as an AMA.

The issue appears once you click the link. Rather than being information about the /z link, it takes you to the phone dial-in instructions link for a specific Zoom meeting. If someone copies the link from that page to share before the meeting, they're sharing a link to a Zoom meeting that won't work later on.

<img width="584" alt="image" src="https://github.com/hackclub/slash-z/assets/76178582/1f0ab354-55d2-47fc-b2b9-bb51e3c09eae">

I'm thinking we could
1. Have an intermediate page with a way to copy the /z link as well as a "Join by phone button"
2. On the dial-in page, we should add a disclaimer that the phone number & link shown are only temporary for that specific meeting

**Comment by JosiasAurel** [comment]
@YodaLightsabr i noticed you self-assigned this issue. Do you want to work on this?

**Comment by YodaLightsabr** [comment]
Yes! I kind of forgot about this with finals & the holidays, but I'll tackle this over the weekend.

**Comment by grymmy** [comment]
@YodaLightsabr reminder on this

**Comment by Prarambha369** [comment]
@YodaLightsabr  reminder on this , Been a Year.
